src/server/server_family.h

```diff
@@ -16,6 +16,11 @@ namespace util {
 class AcceptServer;
 class ListenerInterface;
 class HttpListenerBase;
+
+namespace cloud {
+class AWS;
+}  // namespace cloud
```

Comment:
It's better to refrain from forward declaring classes:
> Avoid using forward declarations where possible. Instead, [include the headers you need](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use).

(Source: https://google.github.io/styleguide/cppguide.html#Forward_Declarations)

Any reason not to include directly?

Reply:
decision: "Try to avoid forward declarations of entities defined in another project."

I use forward declaration when I do not want to expose implementation details and compile cloud utilities by every cc class that includes server_family.